web/views/skill_view.ex

```diff
@@ -0,0 +1,6 @@
+defmodule CodeCorps.SkillView do
+  use CodeCorps.Web, :view
+  use JaSerializer.PhoenixView
+
+  attributes [:title, :description, :original_row, :slug, :inserted_at, :updated_at]
```

Comment:
In the Rails app for tracking imports from the CSV of skills:

`Skill.find_or_create_by(original_row: line["Original Row"])`